addons/sale/models/sale_order.py

```diff
@@ -230,7 +231,7 @@ def _search_invoice_ids(self, operator, value):
     terms_type = fields.Selection(related='company_id.terms_type')
 
     amount_untaxed = fields.Monetary(string='Untaxed Amount', store=True, readonly=True, compute='_amount_all', tracking=5)
-    amount_by_group = fields.Binary(string="Tax amount by group", compute='_amount_by_group', help="type: [(name, amount, base, formated amount, formated base)]")
+    tax_totals_json = fields.Char(compute='_compute_tax_totals_json')
```

Comment:
Honestly, do we even need a field to hold the data ???
Can't it be managed by a simple method called in the xml template ? :thinking: